app/models/rubric_criterion.rb

```diff
@@ -92,28 +93,35 @@ def self.create_or_update_from_csv_row(row, assignment)
     # create a new one.
     criterion = assignment.get_criteria(:all, :rubric).find_or_create_by(name: name)
     # Check that the weight is not a string, so that the appropriate max mark can be calculated.
-    begin
-      criterion.max_mark = Float(working_row.shift) * MAX_LEVEL
-    rescue ArgumentError
-      raise CsvInvalidLineError, I18n.t('upload_errors.invalid_csv_row_format')
-    end
     # Only set the position if this is a new record.
     if criterion.new_record?
       criterion.position = assignment.next_criterion_position
     end
-    # next comes the level names.
-    (0..RUBRIC_LEVELS - 1).each do |i|
-      criterion['level_' + i.to_s + '_name'] = working_row.shift
-    end
-    # the rest of the values are level descriptions.
-    (0..RUBRIC_LEVELS - 1).each do |i|
-      criterion['level_' + i.to_s + '_description'] = working_row.shift
-    end
-    unless criterion.save
-      raise CsvInvalidLineError
-    end
 
-    criterion
+    # there are 5 fields for each level
```

Comment:
Looking at the loop below, it seems like there are 4 fields per level, not 5. (`name`, `number`, `description`, `mark`).